src/backend/gl/Cargo.toml

```diff
@@ -17,10 +17,9 @@ build = "build.rs"
 name = "gfx_backend_gl"
 
 [features]
-#default = ["wgl", "surfman", "surfman-x11"]
-default = []
+default = ["wgl", "surfman", "surfman-x11"]
 wgl = ["winapi"]
-#surfman-x11 = ["surfman/sm-x11"]
+surfman-x11 = ["surfman/sm-x11"]
```

Comment:
We are trying to make default deps as uncontroversial as possible. X11, for example, is not enabled by default in the Vulkan backend - https://github.com/gfx-rs/gfx/blob/379ace36c8ae78c1f2a7e4c5af4e9715b204c1ea/src/backend/vulkan/Cargo.toml#L41

I believe it shouldn't be default here either.
Also, we should call this feature just `x11`, assuming that `surfman` is the main path.